docs/UserGuide.md

```diff
@@ -177,6 +184,6 @@ Action | Format, Examples
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [t/TAG]… [d1/MONDAY] [d2/TUESDAY] [d3/WEDNESDAY] [d4/THURSDAY] [d5/FRIDAY] ​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find [n/KEYWORDS] [p/PHONE_VALUE] [ay/ACADEMIC_YEAR] [e/EMAIL]`<br> e.g., `find n/James Jake ay/2`
+**Find** | `find [n/KEYWORDS] [p/PHONE_VALUE] [ay/ACADEMIC_YEAR] [e/EMAIL] [d1/HHMM d2/HHMM d3/HHMM d4/HHMM d5/HHMM]`<br> e.g., `find n/James Jake ay/2`
```

Comment:
`HHmm` instead? Capital M is for month i think

Reply:
Yeah Andy is right. Hour is `HH` and minute is `mm`. [Source](https://www.w3schools.com/java/java_date.asp)

Reply:
Fixed.